### ISSUE TITLE
Read Automated Decision-Making Article

### ISSUE THREAD

**Issue by ajhaller** [sds-capstone/2022-09-proj7-women-at-table#11]

**Comment by rporta23** [comment]
Christopher Knight (2020) Automated Decision-making and Judicial Review, Judicial Review, 25:1, 21-27, DOI:[ 10.1080/10854681.2020.1732740](https://doi.org/10.1080/10854681.2020.1732740)

**Comment by alejanmg** [comment]
**Automated decision-making** is the process of making a decision by automated means without any human involvement. These decisions can be based on factual data, as well as on digitally created profiles or inferred data. Examples of this include:
- an online decision to award a loan; and
- an aptitude test used for recruitment that uses pre-programmed algorithms and criteria. 

**ISSUES WITH AUTOMATED DECISION-MAKING**
- The use of automated tools and technology and Public Authority decision-making is not an area over which there is an extensive degree of transparency
- There is a **lack of public access to the details** and methods used in automated decision-making end the techniques being used are not currently known about, there is a **lack of public awareness**
-  whenever automated decision-making is being used by public authorities in the exercise of their legal functions important issues of public law can arise
- There is an automatic assumption that automated decision-making will be quicker, more cost-effective, and reduce the risks of human error. In other words, humans will become too reliant on the automated decision making 
- potential for **accidental bias**  (bias in the algorithms caused by the people who made the algorithm)
-  **learned bias** (when the machine learns bias due to patterns in the questions it was asked)

**DUTY OF INQUIRY**
- Principle derived from Secretary of State for Education and Science V. Tameside MBC (court case)
- Duty that falls upon a decision maker to take ‘reasonable steps to acquaint himself with the relevant information in order to enable him to answer the question that he has to answer

**TAMESIDE DUTY: GENERAL PRINCIPLES** 
- Obligation on the decision maker to take such steps to inform himself as are reasonable
-  the public body must decide upon the manner and intensity of inquiry to be undertaken (not the court)
-  the court should not intervene merely because it considers that further inquiries would have been sensible or desirable


**Data Controller’s** job is to carry out a data protection impact assessment. 

- **Impact assessments** are needed when there is: 

      -   Processing on a large scale of special categories
      -   Systematic an extensive evaluation of personal aspects
      -   Systematic monitoring of a publicly accessible area on a large scale. 

- data protection impact assessments are assessed by a data controller. if the court thinks the data controller is lacking in his responsibilities they can call out the data controller but not conduct their own investigation. (failure to meet section 64 obligation)
- In the context of data protection law, the data controller must inform the data subject of 

        -       When it obtains the data from them
        -       The existence of automated decision making
        -       Information about the logic involved in the decision-making process
        -       Significant consequences for the data subject

- Tameside tools force accountability and more (legal) transparency when dealing with data that can and will have an effect on the public. 

**PRECAUTIONARY PRINCIPLES**
- When there is uncertainty about the risks, be proactive and take action before the risks become tangible.
- Answer the question: how much risk is acceptable and how much authority should the public authorities have in fixing that balance?
- Consider that: the risks and seriousness of the harms the AI is intended to address make outweigh the risks and seriousness of the harm the AI may cause
- The more remote the risk the more compelling the evidence that the risk exists should be
- 

**Basically:** 
The judicial review listed many cases and acts and examples of past instances that have curated the laws and principles we currently use for data protection, specifically with Automated Decision-making algorithms. It lists the responsibilities of the people who work with and on these algorithms, discusses the lack of knowledge on this topic, and methods and principles that can be used to alleviate risks while promoting transparency with the data.

**Comment by rporta23** [comment]
read

**Comment by ajhaller** [comment]
Read

**Comment by MargaretBassney** [comment]
read
